DB/src/main/java/io/deephaven/db/v2/sources/UngroupedBoxedBooleanArrayColumnSource.java

```diff
@@ -22,7 +22,7 @@ public Boolean get(long index) {
         long segment = index>>base;
         int offset = (int) (index & ((1<<base) - 1));
         Boolean[] array = innerSource.get(segment);
-        if(offset >= array.length) {
+        if (offset >= array.length || array[offset] == null) {
```

Comment:
@rcaudy should the boolean column null check the array obtained from innerSource here like in the other impls?